extension/parquet/parquet_extension.cpp

```diff
@@ -1448,6 +1484,7 @@ static void ParquetCopySerialize(Serializer &serializer, const FunctionData &bin
 	serializer.WriteProperty(111, "debug_use_openssl", bind_data.debug_use_openssl);
 	serializer.WriteProperty(112, "dictionary_size_limit", bind_data.dictionary_size_limit);
 	serializer.WriteProperty(113, "bloom_filter_false_positive_ratio", bind_data.bloom_filter_false_positive_ratio);
+	serializer.WriteProperty(114, "encoding_compatibility", bind_data.encoding_compatibility);
```

Comment:
Should this be `WritePropertyWithDefault` or `WritePropertyWithExplicitDefault` ?

And most likely also for the above fields, like they are now the plan can't be written by main and read by v1.1.3 (I think).

I would be for moving `dictionary_size_limit` and `bloom_filter_false_positive_ratio` if possible to have a default.

Reply:
All three parameters have a default in `ParquetWriteBindData` so I think this is not a concern

Reply:
I looked into it and we skip writing the value if it is equal to the default. I've added defaults now so that we can skip writing these values in these cases, but I don't think it affects the serialization version.

Reply:
Thanks, I think now looks to make sense to me (the changes between now and v1.1.3 as a whole), but in doubt maybe serialization experts @Maxxen or @Mytherin want to have a peek?